bootcamp/celery.py

```diff
@@ -16,19 +16,27 @@
 
 log = logging.getLogger(__name__)
 
+client = Client(**settings.RAVEN_CONFIG)
 
-class CustomCelery(Celery):
-    """Custom celery class to handle Sentry setup."""
+# register a custom filter to filter out duplicate logs
+register_logger_signal(client)
```

Comment:
The docs I was mentioning are here:
https://docs.sentry.io/clients/python/integrations/celery/

What I do not understand is how in this way the Sentry client is hooked to the Celery app.
In the other way it is much clearer, but I was getting this pylint error:
```
____________________________________________________________________________________________ [pylint] bootcamp/celery.py ____________________________________________________________________________________________
E: 23, 4: An attribute defined in celery.app.base line 296 hides this method (method-hidden)
```